src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private static final String name = "rectangle";
+    private int firstLeg;
+    private int secondLeg;
+    private double area;
+
+    public Rectangle(int firstLeg, int secondLeg, String color) {
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        setColor(color);
+        this.area = getArea();

```

Comment:
what is the point of having an area as a field, and setting it in a constructor if you never access it and for each `draw()` invocation you calculate it again?